libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -73,6 +74,7 @@
 
     private static Map<String, SalesforceAnalyticsManager> INSTANCES;
     private static boolean PUBLISH_HANDLER_ACTIVE;
+    private static ScheduledFuture SCHEDULER;
```

Comment:
Typically, yes, you're right. Static would be something like `sScheduler` as opposed to instance variable being `mScheduler`, but we have been using uppercase all over our projects, so I kept it consistent with that.